determined-ai/determined
# fix: pin icon should adapt to theme colors

## Description





## Test Plan

- Check that the "pin" icon in the design kit adapts to light and dark themes
- Make sure that the Home icon in the navigation sidebar isn't cut off at the bottom





## Commentary (optional)





## Checklist

  See [Release Note](https://github.com/determined-ai/determined/blob/master/docs/release-notes/README.md) for details.

## Ticket
